bin/inference/pycbc_inference

```diff
@@ -102,6 +102,9 @@ parser.add_argument("--nprocesses", type=int, default=None,
     help="Number of processes to use. If not given then use maximum.")
 parser.add_argument("--skip-burn-in", action="store_true", default=False,
     help="Do not burn in with sampler.")
+parser.add_argument("--seed", type=int, default=0,
```

Comment:
What do you want the default setting to be?

If the default is ``0`` then its actually setting the seed to zero so:
```
In [27]: numpy.random.seed(0); numpy.random.randint(40)
Out[27]: 0
In [28]: numpy.random.seed(0); numpy.random.randint(40)
Out[28]: 0
In [29]: numpy.random.seed(0); numpy.random.randint(40)
Out[29]: 0
In [30]: numpy.random.seed(0); numpy.random.randint(40)
Out[30]: 0
In [31]: numpy.random.seed(0); numpy.random.randint(40)
Out[31]: 0
```

Whereas if you use ``None``:
```
In [32]: numpy.random.seed(None); numpy.random.randint(40)
Out[32]: 9
In [33]: numpy.random.seed(None); numpy.random.randint(40)
Out[33]: 12
In [34]: numpy.random.seed(None); numpy.random.randint(40)
Out[34]: 37
In [35]: numpy.random.seed(None); numpy.random.randint(40)
Out[35]: 5
In [36]: numpy.random.seed(None); numpy.random.randint(40)
Out[36]: 35
```

I would assume the ``None`` version was the default if I didn't give a seed. Is this what you meant?